config/monitoring/prometheus.tmpl.yaml

```diff
@@ -0,0 +1,132 @@
+apiVersion: v1
+kind: Template
+metadata:
+  name: prometheus-template
+  annotations:
+    tags: "prometheus,monitoring"
+labels:
+  # will be applied on all objects defined below
+  app: prometheus 
+parameters:
+- name: NAMESPACE
+  description: The namespace in which to install the Prometheus Operator
+  required: true
+- name: SA_NAME
+  description: The name of the local Service Account to use to connect to Prometheus
+  required: true
+
+objects:
+- apiVersion: v1
+  kind: Namespace
+  metadata:
+    name: ${NAMESPACE}
+
+- apiVersion: rbac.authorization.k8s.io/v1
+  kind: ClusterRole
```

Comment:
I need to check if this new Cluster Role is really needed. There may be one that already covers the requirements, since this `kube-rbac-proxy` sidecar is already used in the `openshift-monitoring` namespace